streams/src/test/java/org/apache/kafka/streams/processor/internals/SinkNodeTest.java

```diff
@@ -47,12 +48,10 @@ public void invalidInputRecordTimestampTest() {
 
     @Test(expected = StreamsException.class)
     @SuppressWarnings("unchecked")
-    public void shouldThrowStreamsExceptionOnKeyValyeTypeSerializerMissmatch() {
+    public void shouldThrowStreamsExceptionOnKeyValueTypeSerializerMismatch() {
```

Comment:
Can you update this, and the above test, too -> remove `expected = ...` and use `fail`